doc/smart-switch/ip-address-assigment/smart-switch-ip-address-assignment.md

```diff
@@ -149,14 +175,8 @@ Based on the preset `t1-smartswitch` default topology the configuration generate
             "subtype": "SmartSwitch",
         }
     },
-    "MID_PLANE_BRIDGE" {
```

Comment:
I have a concern with removing the MID_PLANE_BRIDGE table from the config DB. The DHCP server implementation for the Smart Switch depends on this table. Please check this PR: https://github.com/sonic-net/sonic-buildimage/pull/17338
We still can generate the MID_PLANE_BRIDGE table configuration based on the information in the platform.json file.

Reply:
I feel it's strange if we still leave them in the config DB. Because in my current design, we will not support to change the midplane bridge from config DB anymore. If we want to change it, we need to edit the platform.json. A table of Config DB cannot be configured, it's confused.
But I agree to leave it in this PR and I can find folks to update their current implementation if this HLD is approved and merged.

Reply:
To have midplane bridge information in the config DB gives us a more convenient way to read it. However, the configuration should be treated as a read-only.